src/mongocrypt-kms-ctx.c

```diff
@@ -993,8 +1103,17 @@ _mongocrypt_kms_ctx_init_azure_auth (
    char *request_string;
    bool ret = false;
 
+   if (!kms) {
```

Comment:
Suggest converting into a not-null assertion.

Reply:
Changed as suggested.  Will show up in next commit.